budgetportal/tests/test_prov_infra_projects.py

```diff
@@ -49,6 +55,10 @@ def test_import_irm_snapshot(self):
         results = response.data["results"]
         num_of_results = len(results)
         self.assertEqual(num_of_results, 0)
+        self.assertEqual(
+            response.data["csv_download_url"],
+            reverse("provincial-infrastructure-project-api-csv"),
```

Comment:
I would prefer this to be hardcoded so that it's super explicit - otherwise we're just testing that this function is used in the view, and that if the url is changed without meaning to we won't notice (but links to the site might break).

Reply:
Makes sense.